volume_data_converter/erddap_query.py

```diff
@@ -0,0 +1,109 @@
+import os
+from erddapy import ERDDAP
+import json
+import typer
+from halo import Halo
+from pathlib import Path
+
+app = typer.Typer()
+
+configuration_file_path = os.path.join(
+    Path(__file__).absolute().parent, "config", "erddap_configuration.json"
+)
+
+
+@app.command()
+def erddap_query(
+    output_file_path: str = typer.Argument(
+        ..., help="Location where the resuilting .raw files will be saved"
+    ),
+    erddap_configuration_file: str = typer.Option(
+        configuration_file_path,
+        help=" json file with the erddap connection settings",
+    ),
+):
+    """
+    Downloads erddap data according to the settings in the erddap_configuration.json file. Downloads the data and saves it in a NetCDF file
+    Keyword arguments:
+    output_file_path -- Path where the NC file will be saved. It could be a directory or a path to a file with nc extension (.nc)
+    erddap_configuration_file (Optional)-- Path to the errdap_confguration file. By default uses the one installed on this package.
+
+    """
+
+    print(f"configuration_file_path {erddap_configuration_file}")
+    try:
+        conf_file = open(erddap_configuration_file, "r")
+    except OSError:
+        raise (f" Could not open/read file: {erddap_configuration_file} ")
+
+    with conf_file:
```

Comment:
The purpose of this syntax is that the file gets closed after loading, but your `try` statement above is opening the file and it's not closing it, so you'll have a dangling file handler. I think the try/except is unnecessary. If the file doesn't open, you'lll get an error anyways

Reply:
I removed the exception.